scripts/bert/finetune_squad.py

```diff
@@ -425,7 +425,7 @@ def set_new_lr(step_num, batch_id):
 
             step_loss += ls.asscalar()
 
-            if (batch_id + 1) % log_interval == 0:
+            if (batch_id + 1) % (log_interval * accumulate if accumulate else 1) == 0:
```

Comment:
Is this correct? 
```
>>> accumulate = None
>>> log_interval = 10
>>> log_interval * accumulate if accumulate else 1
1
```
should be `log_interval * accumulate if accumulate else  log_interval`?